src/foam/comics/v2/DAOControllerConfig.js

```diff
@@ -24,6 +26,14 @@ foam.CLASS({
   ],
 
   properties: [
+    {
+      class: 'StringArray',
+      name: 'order'
+    },
+    {
+      class: 'Boolean',
+      name: 'orderDesc'
```

Comment:
Better to get rid of this and instead just have an optional '-' sign in front of properties you want to be descending.
Ex.: firstName,lastName,-age